app/src/main/java/com/github/studydistractor/sdp/data/Chat.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.studydistractor.sdp.data
+
+/**
+ * Represent a chat (place where users can exchange messages)
+ *
+ * @param chatId id of the chat
+ * @param messages list of messages that were sent in the chat
+ */
+data class Chat(
+    val chatId: String? = null,
+    val messages: List<Message> = listOf()

```

Comment:
This should be a list of messageIds (we need to flatten firebase) and not a list of messages. There are no foreign keys in firebase realtime so it's kind of mandatory

Reply:
Also it corresponds with what you did with event participants

Reply:
Yup agree will change